examples/webcam/webcam.py

```diff
@@ -36,8 +36,12 @@ def create_local_tracks(play_from):
                     "video=Integrated Camera", format="dshow", options=options
                 )
             else:
-                webcam = MediaPlayer("/dev/video0", format="v4l2", options=options)
-            relay = MediaRelay()
+                webcam = MediaPlayer("/dev/video0", format="v4l2", transcode=transcode, options={"video_size": "1920x1080", "framerate": "30", "input_format": "h264"})
```

Comment:
You probably don't want to merge this as-is since this will be very specific to the webcam being used. LMK if you'd like to bring in my changes and I'll think of a way to pass this in as args.

Reply:
So, your changes are interesting. However is it possible to run `MediaPlayer` get stream as is from camera (without resize, and other operations)? For example, could you running example without `ffmpeg` pipe to file? 

For example, user could configure camera output before to run and then get output stream as is.

Reply:
Yeah that's exactly what this line does --it tells v4l2 to grab the 1920x1080 30fps h264 stream from `/dev/video0`. With your new `transcode` param, no format conversion happens.